src/App.js

```diff
@@ -74,7 +74,7 @@ const App = () => {
         <Timezone timezone={weatherData.timezone} />
         <Clock clockValue={clock.join(':')} />
       </div>
-      <button onClick={getRefresh}>Refresh</button>
+      <button onClick={getRefresh}><a>Refresh</a>  <span></span>  <span></span> </button>
```

Comment:
Nie używaj znacznikow ```<a>``` wewnatrz buttonów. Sam button zapewnia ta funkcjonalnosc

```suggestion
      <button onClick={getRefresh}>Refresh<span></span><span></span></button>
```

Tak samo jesli tutaj potrzebujesz tych spapcji to zrob to w css na marginie niz wstawiac je tutaj w kodzie